test/integration/targets/ec2_instance/inventory

```diff
@@ -0,0 +1,18 @@
+[tests]
+#dummy
```

Comment:
This PR looks great to me. There are a few things that are commented out that could just be removed such as this.